dingo/pipe/data_generation.py

```diff
@@ -154,8 +154,10 @@ def __init__(self, args, unknown_args, create_data=True):
             args.injection_numbers = None
             args.injection_file = None
             args.injection_dict = None
-            args.gaussian_noise = False
             args.injection_waveform_arguments = None
+            args.injection_frequency_domain_source_model = None
+            self._frequency_domain_source_model = None
```

Comment:
Usually we don't modify the private (underscored) attributes directly. I would suggest instead
```suggestion
            self.frequency_domain_source_model = None
```
The `Input` class (which this subclasses eventually) will then set the private attribute.

Was the `injection_frequency_domain_source_model` missing causing something to break in `bilby_pipe=1.3`?

Reply:
Right, at some point it gets called in bilby_pipe with `bilby_pipe==1.3` so I had to set it. Indeed, the change you suggested works as well.